lib/manager/nuget/extract.ts

```diff
@@ -112,6 +113,42 @@ export async function extractPackageFile(
     return { deps };
   }
 
+  if (packageFile.endsWith('global.json')) {
+    const deps: PackageDependency[] = [];
+    let manifest: MsbuildGlobalManifest;
+
+    try {
+      manifest = JSON.parse(content);
+    } catch (err) {
+      logger.debug({ fileName: packageFile }, 'Invalid JSON');
+      return null;
+    }
+
+    if (manifest['msbuild-sdks'] === undefined) {
+      logger.debug(
+        { fileName: packageFile },
+        'This global.json is not a Nuget file'
+      );
+      return null;
+    }
```

Comment:
also validate `sdk.version`.

if it's set, return it as dep with skip-reason as sugessted here: https://github.com/renovatebot/renovate/issues/8558#issuecomment-916731149

Reply:
@viceice I'm sorry, but there is no `skip-reason`, there is `SkipReason`. Maybe you could guide me what exactly `skip-reason` means?